### DISCUSSION TITLE
Missing response body in imported environment

### DISCUSSION THREAD

**Discussion by badrchoubai-beeline** [mockoon/mockoon#828] [Help] [answered]
I am trying to share the exported Mockoon environments I've created with my teammates and noticed that when pulling them from git the response bodies to all of our routes were missing.

**Comment by 255kb** [comment, marked as answer]
There is currently no equivalent between Mockoon features and an OpenAPI file. I suggest that you read [this](https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/).
Some things are not yet supported when you do an export. 
If you want to share your mocks with your teammates, you can [share the Mockoon data file "as is"](https://mockoon.com/docs/latest/mockoon-data-files/sharing-mock-api-files/).

**Reply by badrchoubai-beeline** [reply]
Thank you for the quick reply. I guess I had just assumed that for sharing you'd want to import the environment using 'open environment' and then export it using the 'import/export' menu. But this saves us a step and is now working as expected!
